web/packages/design/src/ResourceIcon/icons.ts

```diff
@@ -16,295 +16,295 @@
-import salesforce from './assets/salesforce.svg';
-import sanity from './assets/sanity.svg';
-import scimDark from './assets/scim-dark.svg';
-import scimLight from './assets/scim-light.svg';
-import securecodewarriorDark from './assets/securecodewarrior-dark.svg';
-import securecodewarriorLight from './assets/securecodewarrior-light.svg';
-import semrush from './assets/semrush.svg';
-import sendgrid from './assets/sendgrid.svg';
-import sentryDark from './assets/sentry-dark.svg';
-import sentryLight from './assets/sentry-light.svg';
-import sequoiaDark from './assets/sequoia-dark.svg';
-import sequoiaLight from './assets/sequoia-light.svg';
-import server from './assets/server.svg';
-import servicenow from './assets/servicenow.svg';
-import shopify from './assets/shopify.svg';
-import sixsenseDark from './assets/sixsense-dark.svg';
-import sixsenseLight from './assets/sixsense-light.svg';
-import skype from './assets/skype.svg';
-import slab from './assets/slab.svg';
-import slack from './assets/slack.svg';
-import snowflake from './assets/snowflake.svg';
-import spaceliftDark from './assets/spacelift-dark.svg';
-import spaceliftLight from './assets/spacelift-light.svg';
-import sparrow from './assets/sparrow.svg';
-import stripeDark from './assets/stripe-dark.svg';
-import stripeLight from './assets/stripe-light.svg';
-import tableau from './assets/tableau.svg';
-import terraform from './assets/terraform.svg';
-import torqDark from './assets/torq-dark.svg';
-import torqLight from './assets/torq-light.svg';
-import troopsAiDark from './assets/troops.ai-dark.svg';
-import troopsAiLight from './assets/troops.ai-light.svg';
-import twilio from './assets/twilio.svg';
-import twitterDark from './assets/twitter-dark.svg';
-import twitterLight from './assets/twitter-light.svg';
-import userorbitDark from './assets/userorbit-dark.svg';
-import userorbitLight from './assets/userorbit-light.svg';
-import validity from './assets/validity.svg';
-import valimail from './assets/valimail.svg';
-import varicentDark from './assets/varicent-dark.svg';
-import varicentLight from './assets/varicent-light.svg';
-import vendr from './assets/vendr.svg';
-import vercelDark from './assets/vercel-dark.svg';
-import vercelLight from './assets/vercel-light.svg';
-import weavegitops from './assets/weavegitops.svg';
-import windowsDark from './assets/windows-dark.svg';
-import windowsLight from './assets/windows-light.svg';
-import wizDark from './assets/wiz-dark.svg';
-import wizLight from './assets/wiz-light.svg';
-import workshopDark from './assets/workshop-dark.svg';
-import workshopLight from './assets/workshop-light.svg';
-import youtube from './assets/youtube.svg';
-import zapier from './assets/zapier.svg';
-import zendeskDark from './assets/zendesk-dark.svg';
-import zendeskLight from './assets/zendesk-light.svg';
-import zoom from './assets/zoom.svg';
-import zoominfo from './assets/zoominfo.svg';
+import activemq from './assets/activemq.svg?no-inline';
```

Comment:
How does `no-inline` work? Or actually, how did it work before?

I do see that this PR reduces the bundle size by a ton. However, if I create an app called "skype" in my v18.0.0-alpha.4 cluster, the browser still sends a request for `web/app/skype.svg`.

Does this suggest that so far the icons were both inlined and returned as separate files anyway?